testdata/BUILD

```diff
@@ -1102,3 +1102,8 @@ bzl_library(
     srcs = ["utils.bzl"],
     visibility = ["//visibility:private"],
 )
+
+filegroup(
```

Comment:
Would it be possible to create an image hash json file? This way we could have `testdata/image-hash.json` which could have:

```
{
  "test_base_with_entrypoint": "..."
}
```

This might be a little easier to use outside of google3 but I'm not sure. WDYT?

Reply:
Can you explain how that would make it easier? These files are only meant to be used in tests, and the test-helper I added in this CL is designed to be generic.  It's essentially just a simple python port of [a generic test helper like this](https://pkg.go.dev/github.com/xorcare/golden#section-readme). I don't see how making it less generic makes it better.